js/local-storage.js

```diff
@@ -0,0 +1,25 @@
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
+
+const saveFormData = () => {
+  const formData = {
+    name: nameInput.value,
+    email: emailInput.value,
+    message: messageInput.value,
+  };
+  localStorage.setItem('formData', JSON.stringify(formData));
+};
+
+const loadFormData = () => {
+  const formData = JSON.parse(localStorage.getItem('formData')) || {};
+  nameInput.value = formData.name || '';
+  emailInput.value = formData.email || '';
+  messageInput.value = formData.message || '';
+};
+
+nameInput.addEventListener('change', saveFormData);
+emailInput.addEventListener('change', saveFormData);
+messageInput.addEventListener('change', saveFormData);
+
+loadFormData();
```

Comment:
Make sure that this is loaded after the page is fully loaded. See https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event

Reply:
Done